src/oic/oic/claims_provider.py

```diff
@@ -96,7 +96,7 @@ def _aggregation(self, info):
         logger.info("RESPONSE: %s" % (sanitize(cresp.to_dict()),))
         return cresp
 
-    #noinspection PyUnusedLocal
+    # noinspection PyUnusedLocal
```

Comment:
I was wondering if we shouldn't just get rid of these 'no qa' comments and fix up the issues they relate to?

@rohe, you probably have something to add there?